reference/forms/types/entity.rst

```diff
@@ -183,6 +183,38 @@ the entity and return an instance of ``QueryBuilder``.
 Overridden Options
 ------------------
 
+choice_name
+~~~~~~~~~~~
+
+.. versionadded:: 2.7
+    The ``choice_name`` option was introduced in Symfony 2.7.
+
+**type**: ``string``, ``callable`` or :class:`Symfony\\Component\\PropertyAccess\\PropertyPath` **default**: id
+
+By default the name of each field is the id of the entity, if it can be read
+from the class metadata by an internal id reader. Otherwise the process will
+fall back on an incremental integer.
+
+choice_value
+~~~~~~~~~~~~
+
+.. versionadded:: 2.7
+    The ``choice_value`` option was introduced in Symfony 2.7.
+
+**type**: ``string``, ``callable`` or :class:`Symfony\\Component\\PropertyAccess\\PropertyPath` **default**: id
+
+As for the ``choice_name`` option, ``choice_value`` use the id by default.
+It allows an optimization in the :class:``Symfony\\Bridge\\Doctrine\\Form\\ChoiceList\\Loader\\DoctrineChoiceLoader`` which will
+only load the ids passed as values while the form submission, preventing all entities,
+even with a custom ``query_builder`` to be loaded.
+If it can be usefull for API to use an entity property as string value, you
+will gain performances by letting this option set by default.
```

Comment:
I think the description for `choice_value` could be reworded a bit to read more fluently. Though I don't have a good suggestion yet.